x/docs/md/intro/getting-started.md

```diff
@@ -14,3 +14,11 @@ Once you have submitted your first exercise, there are several other things you
 1. Browse other solutions to the same problem.
    Provide feedback: ask questions, suggest improvements, and ponder style and readability.
 1. Fetch a new exercise and solve it.
+
+## Command Line Tips 
+
+You can submit a header file along with your implementation using:  
```

Comment:
Can we make this more generic as well?  You can "submit any additional files along with your implementation".

It is not limited to only header files.  And it is not limited to any language tracks.